examples/05_unit_input_data.py

```diff
@@ -92,12 +94,15 @@
             cud = rp.ComputeUnitDescription()
             cud.executable     = '/usr/bin/wc'
             cud.arguments      = ['-c', 'input.dat']
-            cud.input_staging  = ['input.dat']
+            #cud.input_staging  = ['input.dat']
             
           # this is a shortcut for:
-          # cud.input_staging  = {'source': 'client:///input.dat', 
-          #                       'target': 'unit:///input.dat',
-          #                       'action': rp.Transfer}
+            cud.input_staging  = [{'source': 'client:///input.dat', 
+                                   'target': 'unit:///input.dat',
+                                   'action': rp.TARBALL},
+                                  {'source': 'client:///input2.dat', 
+                                   'target': 'unit:///input2.dat',
+                                   'action': rp.TARBALL}]
 
```

Comment:
Wouldn't it make sense to add an additional example?  I would think we should keep the original example on the canonical staging (otherwise we'll need to explain what  TARBALL means...)

Reply:
Done